src/Propel/Generator/Builder/Om/AbstractOMBuilder.php

```diff
@@ -328,7 +328,9 @@ public function declareClassNamespace($class, $namespace = '', $alias = false)
                 //so use the fqcn
                 return ($namespace ? '\\' . $namespace : '') .  '\\' . $class;
             } else {
-                $autoAliasName = 'Child' . $class;
+                // $autoAliasName = 'Child' . $class;
+                // FIXME: USE A DIFFERENT SYSTEM TO GENERATE AN UNIQUE NAME FOR THIS CLASS?
+                $autoAliasName = $class . md5($namespace);
```

Comment:
That just change the aliases generated for class names (for the use in autogenerated classes, doesnt change any class name) I do agree that MD5 is not the best way, but there are not many ways of doing it. At first I though to generate a name based on the namespace and that could probably be better... for example, get the last namespace and concat it to the class name so a case like the one on the PR generates valid use statements that do not collide. That would generate aliases like MinecraftLaboratoriesAccount and ArcLaboratoriesAccount in case of a collision in the class name. Would that be a better approach then?